core/src/main/java/org/apache/any23/rdf/RDFUtils.java

```diff
@@ -564,6 +569,4 @@ public static boolean isAbsoluteIRI(String href) {
         }
     }
 
-    private RDFUtils() {}
```

Comment:
Is this signalling an intention to change the scope of this class to include non-static methods?